docs/security.md

```diff
@@ -1,10 +1,10 @@
 # Security policy
 
-Most of the security aspects of the system will be done via skills and
-skill slots. However for compatibility with the 15.04 snappy
-architecture there is a special skill type called `migration-skill`
+Most of the security aspects of the system will be done via interfaces and
+slots and plugs. However for compatibility with the 15.04 snappy
+architecture there is a special interface type called `old-security`
```

Comment:
From Gustavo's mail, I got that instead of saying "interface type" we should say only "interface".